config/clusters/cloudbank/demo.values.yaml

```
@@ -49,7 +49,7 @@ jupyterhub:
         # Protects against cryptominers - https://github.com/2i2c-org/infrastructure/issues/1216
         # FIXME: This doesn't account for educational institutions that have emails that don't end in .edu,
         # as is the case for some non-euroamerican universities.
-        username_pattern: '^(.+@2i2c\.org|.+\.edu|kalkeab@gmail.com|deployment-service-check)$'
+        username_pattern: '^(.+@2i2c\.org|.+\.edu|.+@gmail.com|kalkeab@gmail.com|deployment-service-check)$'
```

```suggestion
        username_pattern: '^(.+@2i2c\.org|.+\.edu|.+@gmail\.com|kalkeab@gmail\.com|deployment-service-check)$'
```

Not a problem i think, but could be